views/forms/CharFilterForm/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react'
+import {
+  Button,
+  Checkbox,
+  List,
+  ListItem,
+  ListItemText,
+  InputLabel,
+} from '@material-ui/core'
+import Form from '../Form'
+import {
+  FILTER_CATEGORIES,
+  TRUE_STRING,
+  FALSE_STRING,
+} from '../../../constants'
+
+const ChartFilterForm = ({ initialValues, onSubmit, classes }) => {
+  const [values, setValues] = useState(initialValues)
+  const onChange = (target, filterKey) => {
+    setValues({
+      ...values,
+      [filterKey]: values[filterKey].map((filter) =>
+        filter.name === target.name
+          ? {
+              ...filter,
+              value: filter.value === TRUE_STRING ? FALSE_STRING : TRUE_STRING,
+            }
+          : filter
+      ),
+    })
+  }
+
+  return (
+    <Form handleSubmit={(e) => onSubmit(e, values)}>
```

Comment:
do we need to submit the form event or can we just submit the values?

```suggestion
    <Form handleSubmit={() => onSubmit(values)}>
```

Reply:
We need to submit the form event, but I can move it a level up so it's more like. 

`(e) => (e.preventDefault(), onSubmit(formValues)`

Reply:
I think we could still do it as

```tsx
<Form
  handleSubmit={(e) => {
    e.preventDefault()

    return onSubmit(values)
  }}
>
```

This way the parent component doesn't need to know about the event and can just handle the form values.

Reply:
Thank you for this
https://github.com/AMP-SCZ/dpdash/pull/248/commits/145b0275bc16b6a640f0184569937307c95b5ac5